experimenter/experimenter/kinto/tasks.py

```diff
@@ -126,10 +126,14 @@ def handle_rejection(applications, kinto_client):
         if (
             experiment.is_rollout is True
             and experiment.status == NimbusExperiment.Status.LIVE
-            and experiment.status_next == NimbusExperiment.Status.LIVE
+            and (
+                experiment.status_next == NimbusExperiment.Status.LIVE
+                or experiment.status_next == NimbusExperiment.Status.COMPLETE
```

Comment:
you can also write like this ` experiment.status_next == (NimbusExperiment.Status.LIVE or NimbusExperiment.Status.COMPLETE)` like in single line

Reply:
If we are rejecting it in Remote Settings, we want to make sure that we are setting it back to the right state.

For example:
1. We create a normal experiment 
```
status = draft
publish_status = idle
status_next = none
```
2. We request launch. The states change to:
```
status = draft
publish_status = review
status_next = live
```
3. We approve in experiemnter
```
status = draft
publish_status = approve
status_next = live
```
4. It is sent to RS for review
```
status = draft
publish_status = waiting
status_next = live
```
5. We **reject** in RS -- now we have to manually revert `publish_status` and `status_next` back to the values in step 1
```
status = draft
publish_status = IDLE
status_next = NONE
```